docs/src/rules/semi.md

```diff
@@ -76,7 +76,7 @@ This rule has two options, a string option and an object option.
 String option:
 
 * `"always"` (default) requires semicolons at the end of statements
-* `"never"` disallows semicolons as the end of statements (except to disambiguate statements beginning with `[`, `(`, `/`, `+`, or `-`)
+* `"never"` disallows semicolons at the end of statements (except to disambiguate statements beginning with `[`, `(`, `/`, `+`, or `-`)
```

Comment:
I'm not sure if this was a typo. With "as", this means that the rule disallows semicolons that represent the end of a statement, which sounds right. Curious what others think about this.

Reply:
I compared previous statement, and for me it resemble the same goal.

Reply:
To me, both *as* and *at* sound okay here. Considering that "at the end" is used in the rest of the file without exceptions, I tend to believe that this was indeed an oversight.

Reply:
Okay, let's switch to "at" for consistency.